pkgs/by-name/so/sobjectizer/package.nix

```diff
@@ -0,0 +1,58 @@
+{
+  pkgs,
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+  cmake,
+  withStatic ? true,
+  withShared ? true,
+  buildExamples ? false,
+  buildTests ? true,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "sobjectizer";
+  version = "5.8.4";
+
+  src = fetchFromGitHub {
+    owner = "Stiffstream";
+    repo = "sobjectizer";
+    tag = "v.${finalAttrs.version}";
+    hash = "sha256-tIqWgd6TppHfqZk3XHzhG0t+Nn8BQCTP81UD7ls67UE=";
+  };
+
+  patches = [
+    # Tests do not require the static library.
+    (fetchpatch {
+      url = "https://github.com/Stiffstream/sobjectizer/commit/10eb34c65ccdaa4fea62d0c4354b83104256370d.patch";
```

Comment:
You should use `url` https://github.com/Stiffstream/sobjectizer/commit/8903cb6ab0547e8914506105dd97e6564aea9fc1.patch, as the commit you're pointing to can disappear, should the history be rewritten, whereas the one I recommended is attached to the PR, and cannot disappear

Reply:
You should also add a simple `name` here